tsconfig.json

```diff
@@ -47,6 +47,9 @@
       "@archetypes/*": [
         "archetypes/*"
       ],
+      "~/constants/*": [
```

Comment:
how come `~` over `@`? it is an odd one out now

Reply:
am going through and changing all of them